lib/v2/jsu/rjxy.js

```diff
@@ -0,0 +1,43 @@
+// 导入必要的模组
+const got = require('@/utils/got'); // 自订的 got
+const cheerio = require('cheerio'); // 可以使用类似 jQuery 的 API HTML 解析器
+const { parseDate } = require('@/utils/parse-date');
+const { getPageItem } = require('@/v2/jsu/utils');
+
+module.exports = async (ctx) => {
+    const baseUrl = 'https://rjxy.jsu.edu.cn/';
+
+    const response = await got({
+        method: 'get',
+        url: 'https://rjxy.jsu.edu.cn/index/tzgg1.htm',
+    });
+
+    const $ = cheerio.load(response.data);
+    const list = $('body > div.list-box > div > ul > li').toArray();
+    const out = await Promise.all(
+        list.map(async (item) => {
+            item = $(item);
+            const link = item.find('a').attr('href').replace('../', baseUrl);
```

Comment:
You can use [WHATWG URL API](https://nodejs.org/api/url.html#new-urlinput-base) to resolve related link.